### PR TITLE
Added news content, redirects, 404 error, and other fixes

### PR DESCRIPTION
**Pull request by jhung**
Added:
- news content
- 404 error page
- redirects for some previously used URLs
- favicon
- and more.

### CODE REVIEW

**Comment by jhung** [comment]
The best way to test this is to test locally using 'docpad run' as it will avoid the path issues documented in FLUID-5588 and FLUID-5590.

File: src/documents/404.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+permalink: /404.html
+---
+<main class="fluid-web-about" id="fluid-web-content">
+    <div class="row">
+        <h2>Page not found</h2>
+        <p>
+            We recently updated this website and we are in the process of restoring some lost content.
+        </p>
+        <p>
+            In the meantime, please visit the <a href="http://fluidproject.org/">Fluid Project home page</a>.
```
Comment (acheetham) [inline src/documents/404.html]: I wonder if we might want this link to be relative, i.e. <a href="/">, as the fluid link in the header is.

File: src/documents/news/2012-10-03-We_are_looking_for_a_senior_designer.html.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+---
+*Are you creative, original, and willing to take risks?*
+
+Lead Designer, Inclusive Design Research Centre, OCAD University, Toronto, ON
+
+Keywords: user-centered design, participatory design, inclusive design, user interactions, use-cases, designing for diversity, wireframes, high fidelity mockups, design process.
+
+Are you creative, original, and willing to take risks? Do you want to radically improve user interfaces? Do you like working with others, learning, and teaching? Are you interested in designing for diversity rather than the typical?
+
+The IDRC is a research and development centre focusing on ensuring that emerging information technology and practices are designed inclusively � making sure that the delight and power of the internet is extended to everyone.
```
Comment (acheetham) [inline src/documents/news/2012-10-03-We_are_looking_for_a_senior_designer.html.md]: Weird character not rendering nicely.

**Comment by acheetham** [comment]
The blog postings are now present in the repository but not rendered anywhere, is that right?

File: src/files/js/uio.js [hunk 3]
```diff
@@ -1,14 +1,17 @@
     $(document).ready(function () {
+
+        /* FLUID-5590: should not use window.location.origin since it will break with non-root installations */
```
Comment (acheetham) [inline src/files/js/uio.js]: If this technique is not the ideal technique, we might want to expound a bit in the comment on what we're trying to accomplish and why.

File: src/partials/header.html.eco [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    <nav  id="fluid-web-navigation" class="small-12 medium-8 columns fluid-web-nav">
+        <ul class="right button-group">
+            <% for page in @getCollection("sitenav").toJSON(): %>
+                <li class="">
+                    <a href="<%= @site.url %>/<%= page.name %>" class="button <%= if page.id is @document.id then 'current' else 'inactive' %>">
+                        <%= page.navigationSlug %>
+                    </a>
+                </li>
+            <% end %>
+        </ul>
```
Comment (acheetham) [inline src/partials/header.html.eco]: The navigation links in the upper right corner are now positioned slightly differently: they're higher up than they were. It looks like previously, the bottom of the links was lined up with the bottom of the fluid logo. Now, the tops seem lined up. Not a big deal, but if it's an easy fix...

**Comment by jhung** [comment]
Blog posts are not yet part of this. Just news posts.

File: src/partials/header.html.eco (hunk 4 above)
Comment (jhung) [inline src/partials/header.html.eco]: Thanks. I've updated this.

**Pull request merged**